tests/virtualization/vagrant/add_box.pm

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test for vagrant and packaged addons
+# Maintainer: dancermak <dcermak@suse.com>
```

Comment:
ah, one more thing. Could you add a line with `# Tags:` referencing a ticket or bug or any reference to the motivation why we have that test?

Please also reference that ticket in the commit message and/or PR description

Reply:
Uh, I don't really have a bugreport or a ticket as a motivation besides: "vagrant broke multiple times in the past half year and I need it working". Also I've used this as a preparation to create a openQA test for the new openSUSE Tumbleweed vagrant box.